pkg/kubectl/cmd/templates/command_groups.go

```diff
@@ -59,3 +66,25 @@ func AddAdditionalCommands(g CommandGroups, message string, cmds []*cobra.Comman
 	}
 	return append(g, group)
 }
+
+func CheckRequiredFlags(cmd *cobra.Command, _ []string) error {
+	flags := cmd.Flags()
+	missingFlagNames := []string{}
+
+	flags.VisitAll(func(flag *pflag.Flag) {
+		requiredAnnotation, found := flag.Annotations[cobra.BashCompOneRequiredFlag]
```

Comment:
@liggitt Seems most of current commands don't follow this design convention. It would be troublesome to add this validation for every commands, even subcommands.